src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/model/caserecord/SupplementaryEvidenceRecord.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.model.caserecord;
+
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.model.CaseRecord;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.model.Envelope;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDataContent;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.ccd.client.model.Classification;
+
+public class SupplementaryEvidenceRecord implements CaseRecord {
```

Comment:
There is nothing called as SupplementaryEvidenceRecord either it is an exception or case